### PR TITLE
reduced logging verbosity

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

File: .travis/run.sh [hunk 1]
```diff
@@ -13,7 +13,6 @@ coverage run \
     -m py.test \
     -Wd \
     --travis-fold=always \
-    -vvvvvv \
```
Comment (palango) [inline .travis/run.sh]: Should we leave some verbosity?

Comment (pcppcp) [reply .travis/run.sh]: I think it'd be sufficient to just remove geth or py-evm logs which are quite verbose. But personally I almost never read Travis logs output so I don't care.

Comment (hackaugusto) [reply .travis/run.sh]: it's the next line `--log-config='raiden:DEBUG' ` it enables debug logging for raiden.

this is disabling the `DEBUG` level for everything that is not raiden [1](https://github.com/raiden-network/raiden/blob/master/raiden/tests/conftest.py#L152-L153), and the [geth verbosity](https://github.com/raiden-network/raiden/blob/master/raiden/tests/utils/blockchain.py#L94)

**Comment by andrevmatos** [comment]
@pcppcp any way to maybe supress JSON [stdout] output from structlog, and write normal raiden logging to stdout instead of stderr (as well as ensuring geth logs also goes to stderr)? With that, we could put "less useful" (geth, synapse) logging to `stderr` and raiden debug logging to `stdout`, and make use of pytest/travis logs folding to better view what matters, going to the sinkhole when needed.

**Comment by hackaugusto** [comment]
we can forward the logs from synapse to stderr if you want, but geth logs are not helpful for debugging our protocol at all, I really see no point in keeping it

**Comment by andrevmatos** [comment]
Ok, I don't hold love for geth logs either.. synapse logs already go to stderr by default, that's why I suggested moving main raiden logs to stdout to split it.

**Comment by ulope** [comment]
@andrevmatos Ideally the structlog messages should go neither to stdout nor stderr but be handled by pytest's logging integration. I'll take a look at that (been annoyed by that for a while now).

**Pull request merged**